packages/@aws-cdk/aws-redshift-alpha/README.md

```diff
@@ -76,6 +76,29 @@ const cluster = new Cluster(this, 'Redshift', {
 });
 ```
 
+## Availability Zone Relocation
+
+By using [relocation in Amazon Redshift](https://docs.aws.amazon.com/redshift/latest/mgmt/managing-cluster-recovery.html), you allow Amazon Redshift to move a cluster to another Availability Zone (AZ) without any loss of data or changes to your applications.
+
+To enable this feature, set the `availabilityZoneRelocation` property to `true` when creating the cluster.
```

Comment:
Just to clarify, relocation cannot be enabled after a cluster has been created?

It appears from the current README that relocation cannot be enabled after a cluster is created, but the documentation suggests that it should be possible to enable it post-creation.

https://docs.aws.amazon.com/redshift/latest/mgmt/managing-cluster-recovery.html#cluster-recovery-console

Reply:
You're absolutely right. I've updated description.

```md
By using [relocation in Amazon Redshift](https://docs.aws.amazon.com/redshift/latest/mgmt/managing-cluster-recovery.html), you allow Amazon Redshift to move a cluster to another Availability Zone (AZ) without any loss of data or changes to your applications.
This feature can be applied to both new and existing clusters.

To enable this feature, set the `availabilityZoneRelocation` property to `true`.
```